developerDocs/advanced-use-cases.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Advanced Use Cases
+category: 64cbb5277b5f3c0065d96616
+slug: opensea-sdk-advanced-use
+parentDocSlug: opensea-sdk
+order: 2
+hidden: false
+---
+
+- [Scheduling Future Listings](#scheduling-future-listings)
+- [Purchasing Items for Other Users](#purchasing-items-for-other-users)
+- [Using ERC-20 Tokens Instead of Ether](#using-erc-20-tokens-instead-of-ether)
+- [Private Auctions](#private-auctions)
+- [Listening to Events](#listening-to-events)
+
+## Advanced
+
+Interested in purchasing for users server-side or with a bot, scheduling future orders, or making bids in different ERC-20 tokens? OpenSea.js can help with that.
+
+### Scheduling Future Listings
+
+You can create sell orders that aren't fulfillable until a future date. Just pass in a `listingTime` (a UTC timestamp in seconds) to your SDK instance:
+
+```typescript
+const auction = await openseaSDK.createSellOrder({
```

Comment:
isn't this a listing not an auction? english auction would have to use `waitForHighestBid: true` from my understanding

Reply:
Yeah, whenever this readme was written they were calling all sell orders auctions